src/components/BusinessProductsListing/index.js

```diff
@@ -62,10 +63,12 @@ const BusinessProductsListingUI = (props) => {
     errorQuantityProducts
   } = props
 
+  const [{ configs }] = useConfig()
+  const { add_product_with_one_click } = configs
```

Comment:
import this config like that:

`const addProductWithOneClick = configs?.addProductWithOneClick?.value`